### PR TITLE
updated German translation

### PR DESCRIPTION
**Pull request by Danfro**
/qml/phoneui/iu/PasswordPage.qml line 101 
The dot at the end of the string ""This will permanently delete all messages!." wants to be deleted, I think.
The header of the linked devices page is labeled "link a device", not "linked devices".

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -108,34 +112,30 @@ msgid "Enter key sends"
...
 msgid "Activate Debuglog"
-msgstr "Diagnoseprotokoll übermitteln"
+msgstr "Diagnoseprotokoll aktivieren"
 
 #: ../qml/phoneui/ui/settings/LinkedDevicesPage.qml:13
 #: ../qml/phoneui/ui/settings/LinkedDevicesPage.qml:17
 #: ../qml/phoneui/ui/settings/AddLinkedDevicesPage.qml:10
 msgid "Link a Device"
-msgstr ""
+msgstr "Gerät verknüpfen"
```
Comment (RobertZenz) [inline po/de.po]: I'd rather use "Geraet hinzufuegen" or "Geraet verbinden". "Verknuepfen" is rather seldom used in this context, eve though "link" would translate to "verknuepfen".

File: po/de.po [hunk 2]
```diff
@@ -1,64 +1,64 @@
...
 #: ../qml/phoneui/ui/SetPasswordPage.qml:17
 #: ../qml/phoneui/ui/SettingsPage.qml:51
 msgid "Create passphrase"
-msgstr ""
+msgstr "Passwort anlegen"
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:53
 msgid "Old passphrase"
-msgstr ""
+msgstr "Bisheriges Passwort"
```
Comment (RobertZenz) [inline po/de.po]: I'd go for "Alt" here.

File: po/de.po [hunk 3]
```diff
@@ -1,64 +1,64 @@
...
 
 #: ../qml/phoneui/ui/SetPasswordPage.qml:71
-#, fuzzy
 msgid "New passphrase"
-msgstr "Neue Nachricht"
+msgstr "Neues Passwort hier eingeben"
```
Comment (RobertZenz) [inline po/de.po]: I'd drop the "hier", as it adds nothing.

File: po/de.po [hunk 4]
```diff
@@ -199,16 +199,16 @@ msgstr "Hilfe"
...
-#, fuzzy, qt-format
+#, qt-format
 msgid "Forward message to %1?"
-msgstr "Nachricht weiterleiten"
+msgstr "Nachricht weiterleiten an %1?"
 
 #: ../qml/phoneui/ui/SettingsPage.qml:59
 msgid "Linked Devices"
-msgstr ""
+msgstr "Verknüpfte Geräte"
```
Comment (RobertZenz) [inline po/de.po]: Also I'd go here with "Verbundene Geraete".

File: po/de.po (hunk 1 above)
Comment (Danfro) [reply po/de.po]: Sounds good. "Gerät verbinden" would be fine with me.

File: po/de.po (hunk 4 above)
Comment (nanu-c) [reply po/de.po]: I like also "Verbundene Geräte"

**Comment by nanu-c** [comment]
I will merge if you resolve the reviews :)

**Comment by Danfro** [comment]
Uhm. Should I have changed the PR somehow to match the above discussion (verknüpft --> verbunden) before resolving?
Or are you going to do that manually in your files?

Spotted one more typo in de.po:
`#: ../qml/phoneui/ui/SettingsPage.qml:106
msgid "Activate Debuglog"
msgstr "Dubug-Modus aktivieren"`
Of course it wants to be "Debug-Modus aktivieren"., not Dubug. ;-)

**Pull request merged**